packages/server/graphql/private/mutations/startTrial.ts

```diff
@@ -0,0 +1,46 @@
+import removeTeamsLimitObjects from '../../../billing/helpers/removeTeamsLimitObjects'
+import getRethink from '../../../database/rethinkDriver'
+import getKysely from '../../../postgres/getKysely'
+import setTierForOrgUsers from '../../../utils/setTierForOrgUsers'
+import setUserTierForOrgId from '../../../utils/setUserTierForOrgId'
+import hideConversionModal from '../../mutations/helpers/hideConversionModal'
+import {MutationResolvers} from '../resolverTypes'
+
+const startTrial: MutationResolvers['startTrial'] = async (_source, {orgId}, {dataLoader}) => {
+  const r = await getRethink()
+  const pg = getKysely()
+  const now = new Date()
+  const organization = await dataLoader.get('organizations').load(orgId)
+
+  // VALIDATION
+  if (organization.tier !== 'starter') {
+    throw new Error('Cannot start trial for organization on paid tier')
```

Comment:
-1 You've declared an `ErrorPayload` as a valid result, so we should not throw here

Reply:
Fixed